fearless/Common/Storage/EntityToModel/AssetModelMapper.swift

```diff
@@ -10,17 +10,27 @@ final class AssetModelMapper: CoreDataMapperProtocol {
     var entityIdentifierFieldName: String { "id" }
 
     func transform(entity: CDAsset) throws -> AssetModel {
-        guard let id = entity.id, let chainId = entity.chainId else {
+        guard
+            let id = entity.id,
+            let chainId = entity.chainId,
+            let symbol = entity.symbol,
+            let typeRawValue = entity.type
```

Comment:
in AssetModel constructor you try to create ChainAssetType for given typeRawValue and provide default value (.normal) in case of fail. so it looks like optional field